test/e2e/test/elasticsearch/checks_es.go

```diff
@@ -149,17 +137,19 @@ func (e *esClusterChecks) CheckESNodesTopology() test.Step {
 				// match the actual Elasticsearch node to an expected one in expectedTopology
 				nodeStats := nodesStats.Nodes[nodeID]
 				var err error
+				var allErrors []string
 				for i, topoElem := range expectedTopology {
 					if err = e.compareTopology(es, topoElem, node, nodeStats); err == nil {
 						// found it! no need to match this topology anymore
 						expectedTopology = append(expectedTopology[:i], expectedTopology[i+1:]...)
 						break
 					}
+					allErrors = append(allErrors, fmt.Sprintf("%s: %s", topoElem.Name, err.Error()))
```

Comment:
Nit: remove the `topoElem.Name` as it is already in the error returned by compareTopology?
```suggestion
					allErrors = append(allErrors, fmt.Sprintf("%s", err.Error()))
```
Otherwise it's a bit redundant:
```
actual node node1 in the cluster does not match any expected nodes: [nodeSet1: node node1 does not \
belong to nodeSet nodeSet1 nodeSet2: node node1 does not belong to nodeSet nodeSet2]
```

Reply:
Actually this is only true for the error I added the others are lacking the context. I will remove it from the new error on L168.